docs/source/amp.rst

```diff
@@ -29,7 +29,7 @@ datatype of ``torch.bfloat16`` only uses :class:`torch.autocast`.
     ``torch.cuda.amp.autocast(args...)`` and ``torch.cpu.amp.autocast(args...)`` will be deprecated. Please use ``torch.autocast("cuda", args...)`` or ``torch.autocast("cpu", args...)`` instead.
     ``torch.cuda.amp.GradScaler(args...)`` and ``torch.cpu.amp.GradScaler(args...)`` will be deprecated. Please use ``torch.GradScaler("cuda", args...)`` or ``torch.GradScaler("cpu", args...)`` instead.
 
-:class:`torch.autocast` and :class:`torch.cpu.amp.autocast` are new in version `1.10`.
```

Comment:
Let us keep this as it is because the deprecation warning has existed on line 29-30.